libraries/SITL/SIM_BalanceBot.h

```diff
@@ -0,0 +1,42 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  BalanceBot simulator class
+*/
+
+#pragma once
+
+#include "SIM_Rover.h"
+
+namespace SITL {
+
+class BalanceBot : public SimRover {
```

Comment:
I think it would be better to not inherit from SimRover, and instead make it standalone